puppet/zulip/templates/uwsgi.ini.template.erb

```diff
@@ -11,3 +10,4 @@ post-buffering=4096
 env= LANG=en_US.UTF-8
 uid=zulip
 gid=zulip
+directory=/home/zulip/deployments/current/
```

Comment:
@rht what's the story with this change?  It seems unrelated to the rest, so should have its own commit with its own commit message explaining why it is correct.